src/vm/memory/relocatable.zig

```diff
@@ -438,6 +610,42 @@ test "Relocatable: addMaybeRelocatableInplace should return an error if other is
     );
 }
 
+test "Relocatable: lt should return true if other relocatable is greater than or equal, false otherwise" {
+    try expect(Relocatable.new(-1, 2).lt(Relocatable.new(-1, 3)));
+    try expect(Relocatable.new(1, 5).lt(Relocatable.new(2, 4)));
+
+    try expect(!Relocatable.new(2, 4).lt(Relocatable.new(2, 4)));
+    try expect(!Relocatable.new(2, 5).lt(Relocatable.new(2, 4)));
+    try expect(!Relocatable.new(3, 3).lt(Relocatable.new(2, 4)));
+}
+
+test "Relocatable: le should return true if other relocatable is greater, false otherwise" {
+    try expect(Relocatable.new(-1, 2).le(Relocatable.new(-1, 3)));
+    try expect(Relocatable.new(1, 5).le(Relocatable.new(2, 4)));
+    try expect(Relocatable.new(2, 4).le(Relocatable.new(2, 4)));
+
+    try expect(!Relocatable.new(2, 5).le(Relocatable.new(2, 4)));
+    try expect(!Relocatable.new(3, 3).le(Relocatable.new(2, 4)));
+}
+
+test "Relocatable: gt should return true if other relocatable is less than or equal, false otherwise" {
+    try expect(!Relocatable.new(-1, 2).gt(Relocatable.new(-1, 3)));
+    try expect(!Relocatable.new(1, 5).gt(Relocatable.new(2, 4)));
+    try expect(!Relocatable.new(2, 4).gt(Relocatable.new(2, 4)));
+
+    try expect(Relocatable.new(2, 5).gt(Relocatable.new(2, 4)));
+    try expect(Relocatable.new(3, 3).gt(Relocatable.new(2, 4)));
+}
+
+test "Relocatable: ge should return true if other relocatable is less, false otherwise" {
+    try expect(!Relocatable.new(-1, 2).gt(Relocatable.new(-1, 3)));
```

Comment:
Here I think that you wanted to test `ge` instead of `gt` no?

Reply:
my bad, copy pasted the test and forgot to update the body xP